nupic/regions/AnomalyRegion.py

```diff
@@ -78,6 +78,17 @@ def __init__(self, *args, **kwargs):
     self.prevPredictedColumns = numpy.zeros([], dtype="float32")
 
 
+  @classmethod
+  def read(cls, proto):
+    anomalyRegion = object.__new__(cls)
+    anomalyRegion.prevPredictedColumns = numpy.array(proto.prevPredictedColumns)
+    return anomalyRegion
+
+
+  def write(self, proto):
+    proto.prevPredictedColumns = self.prevPredictedColumns.tolist()
```

Comment:
Does this work? Don't you need to first init the proto list and then set it?

http://jparyani.github.io/pycapnp/quickstart.html#list